api/v1beta1/frrconfiguration_types.go

```diff
@@ -143,8 +143,9 @@ type Neighbor struct {
 	BFDProfile string `json:"bfdProfile,omitempty"`
 
 	// EnableGracefulRestart allows BGP peer to continue to forward data packets along
-	// known routes while the routing protocol information is being restored.
-	// This field is immutable because it requires restart of the BGP session
+	// known routes while the routing protocol information is being restored.  If
```

Comment:
I'd reword the warning with: "if the session is already established, the configuration will have effect after reconnecting to the peer."

Reply:
done

Reply:
There's something odd, I still see the old description here as comment but hte new one in the generated crds (while it should be the same, I think?)